cirq-core/cirq/experiments/two_qubit_xeb.py

```diff
@@ -373,6 +375,7 @@ def parallel_xeb_workflow(
         random_state: The random state to use.
         ax: the plt.Axes to plot the device layout on. If not given,
             no plot is created.
+        pool: An optional multiprocessing pool.
```

Comment:
Can we set a default value for `pool` so that it runs in parallel by default?

Reply:
done